src/wallet/rpc/coins.cpp

```diff
@@ -699,7 +699,7 @@ RPCHelpMan listunspent()
                 entry.pushKV("ancestorfees", uint64_t(ancestor_fees));
             }
         }
-        entry.pushKV("spendable", out.spendable);
+        entry.pushKV("spendable", true); // Any coins we list are always spendable
```

Comment:
Would removing this property from the response categorise the diff as breaking change?
Looks a little odd to me to display this as `true` for watch only wallets as well because the wallet doesn't have private keys and thus I feel it should not be a decision maker on this.

Reply:
> Would removing this property from the response categorise the diff as a breaking change?

Yes, that's mainly why this is preserved.

>  Looks a little odd to me to display this as true for watch only wallets as well because the wallet doesn't have private keys and thus I feel it should not be a decision maker on this.

This is the current behavior anyways.